src/views/auth/forgot-password.vue

```diff
@@ -0,0 +1,82 @@
+<template>
+    <auth-container>
+        <template slot="auth-form">
+            <form
+                id="form-login"
+                class="p-t-15"
+                autocomplete="on"
+                @submit.prevent="verifyFields()">
+                <div class="form-group form-group-default required">
+                    <label>{{ form.data.email.label }}</label>
+                    <div class="controls">
+                        <input
+                            v-validate="validations.email"
+                            v-model="data.email"
+                            type="text"
+                            name="email"
+                            autocomplete="on"
+                            placeholder="user@example.com"
+                            class="form-control"
+                            required>
```

Comment:
Due the `data.email.validations` has the `required` attribute this is not necessary
```suggestion
>
```